addons/auth_signup/data/mail_template_data.xml

```diff
@@ -93,17 +92,17 @@
 </td></tr>
 </table>
             </field>
-            <field name="lang">${object.lang}</field>
+            <field name="lang">{{ object.lang }}</field>
             <field name="auto_delete" eval="True"/>
         </record>
 
         <!-- Email template for new users -->
         <record id="set_password_email" model="mail.template">
             <field name="name">Auth Signup: Odoo Connection</field>
             <field name="model_id" ref="base.model_res_users"/>
-            <field name="subject">${object.create_uid.name} from ${object.company_id.name} invites you to connect to Odoo</field>
-            <field name="email_from">"${object.company_id.name | safe}" &lt;${(object.company_id.email or user.email) | safe}&gt;</field>
-            <field name="email_to">${object.email_formatted | safe}</field>
+            <field name="subject">{{ object.create_uid.name }} from {{ object.company_id.name }} invites you to connect to Odoo</field>
+            <field name="email_from">"{{ object.company_id.name }}" &lt;{{ (object.company_id.email or user.email) }}&gt;</field>
```

Comment:
Would also update as stated above (separate commit as this is an improvement/fix)